### DISCUSSION TITLE
Unwanted Space Adds Between Code Block and Bulleted Paragraph

### DISCUSSION THREAD

**Discussion by Ashkaan** [requarks/wiki#3341] [Error / Bug Report]
### Actual behavior
There is an additional space between a CODE BLOCK and a BULLETED PARAGRAPH after I save using the visual editor.  If I go to edit, the space is indeed there.  If I delete it and save, it looks good, but then when I refresh the space is back!
### Expected behavior
If I ensure that there is no space in the visual editor, then when I save the space should not be there.
### Steps to reproduce the behavior
Write something inside of a CODE BLOCK and then write something inside of BULLETED PARAGRAPH immediately after without a space and save the page.  Exit the editor and you will see an extra unwanted space.

<img width="410" alt="Screen Shot 2021-03-12 at 12 05 24 PM" src="https://user-images.githubusercontent.com/5667978/110992791-4470af00-832b-11eb-8bf6-2bbd676897fb.png">

OS: Docker
Wiki.js: 2.5.170
Database: Postgres (latest docker)

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

**Comment by Ashkaan** [comment]
> You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

Fixed

**Comment by Ashkaan** [comment]
Hi, I'm just checking to see if anyone has visibility on this.  Thanks!
